brunoise/streaming_save.py

```diff
@@ -63,13 +66,22 @@ def save_loop(self):
         (Path(self.save_parameters.output_dir) / "original").mkdir(
             parents=True, exist_ok=True
         )
+        if self.save_parameters.channel == "Both":
```

Comment:
Should there not be a case for "green" and "red" as well?

Reply:
This is fine. I think some of our analysis pipelines assume this structure. The reason to save green and red separately is because `SplitDataset` doesn't support multi-channel. `complete_plane` handles the actual saving process.